client-runtime/src/main/resources/META-INF/resources/webjars/stjs-client-runtime/stjs.js

```diff
@@ -140,13 +140,24 @@ if (!String.prototype.replaceFirst){
 }
 
 String.prototype.indexOf$String=function(str) {
-    return this.indexOf(str) >= 0;
+    return this.indexOf(str);
+}
+
+String.prototype.indexOf$String_int=function(str, pos) {
+    return this.indexOf(str, pos);
 }
 
-String.prototype.indexOf$String_int=stjs.NOT_IMPLEMENTED;
 String.prototype.indexOf$int=stjs.NOT_IMPLEMENTED;
 String.prototype.indexOf$int_int=stjs.NOT_IMPLEMENTED;
 
+String.prototype.substring$int=function(startIndex) {
+    return this.substring(startIndex, this.length);

```

Comment:
`this.length` n’est pas nécéssaire ici, c'est ça par défaut si tu ne passe pas le 2ème paramètre.